humanhash.py

```diff
@@ -139,31 +140,38 @@ def compress(bytes_, target):
 
             >>> bytes_ = [96, 173, 141, 13, 135, 27, 96, 149, 128, 130, 151]
             >>> list(HumanHasher.compress(bytes_, 4))
-            [205, 128, 156, 96]
+            [64, 145, 117, 21]
 
-        Attempting to compress a smaller number of bytes to a larger number is
-        an error:
+        If there are less than the target number bytes, return input bytes
 
-            >>> HumanHasher.compress(bytes_, 15)  # doctest: +ELLIPSIS
-            Traceback (most recent call last):
-            ...
-            ValueError: Fewer input bytes than requested output
+            >>> list(HumanHasher.compress(bytes_, 15))  # doctest: +ELLIPSIS
+            [96, 173, 141, 13, 135, 27, 96, 149, 128, 130, 151]
         """
 
         bytes_list = list(bytes_)
 
         length = len(bytes_list)
-        if target > length:
-            raise ValueError("Fewer input bytes than requested output")
```

Comment:
I removed this check to make the function more generous in what it accepts. I changed the comments to reflect the change. My thinking is that the purpose of compression is to ensure that there are no more than the target number of bytes, which is satisfied if target > length. So now if target > length, the original bytes are returned since there is no compression needed. 

I do understand that this is a different behavior. The impact on end users is that hashing small inputs will not cause an error, which I think is more intuitive. Another solution would be to pad input bytes up to the target number, but this would lead to padded final outputs.

For example:
```python
digest = '010203'

# Old method
humanhash.humanize(digest, 6)
Traceback (most recent call last):
...
ValueError: Fewer input bytes than requested output

# New method
humanhash.humanize(digest, 6)
'alabama-alanine-alaska'

# Padding method
humanhash.humanize(digest, 6)
'alabama-alanine-alaska-ack-ack-ack'
```

It's your call, I'm happy to help if I can. :)